### DISCUSSION TITLE
5.1.0 release notes

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-Lern/Community#70] [Documentation]
@rhwarrier @reshmi-nair - as per the microsite, 5.1.0 should be live by 20th Jan. However, I could not locate any upgrade instructions or release notes in the microsite. Please share us these details. ED team's cut-off is 3rd Feb to check & integrate the latest prod versions from other BBs

@vinukumar-vs @gandham-santhosh

**Comment by rhwarrier** [comment]
@Krishnaj20  the release notes and the tag will be published today.
cc : @reshmi-nair

**Reply by reshmi-nair** [reply]
@Krishnaj20 Pls find the release notes for 5.1.0 [here](https://app.gitbook.com/o/-Mi9QwJlsfb7xuxTBc0J/s/4ZKyfmmhMWpPkD6iYvKF/use/release-notes/release-v-5.1.0)
cc: @rhwarrier

**Reply by Krishnaj20** [reply]
@reshmi-nair @rhwarrier - we have gone thru the Lern release notes but we are unable to find the details of the changes done by the team. For example, LR-6 & LR-131 we have no clue about these changes - such as tech documentation, test cases etc - for the ED team to do the impact analysis before taking up this version of Lern. Please share these artefacts for the 5.1.0 release. 

@vinukumar-vs @chitranshu-keshav

**Reply by reshmi-nair** [reply]
@Krishnaj20 LR-6 will be removed from the release notes as we could not complete the testing on it. LR-131 we have some code clean up activity which doesnot have any other service dependency. I will update the release notes with the same

**Comment by Hari-stackroute** [comment]
A hot-fix is released for data-pipeline jobs(batch-service related) for Jira bug: https://project-sunbird.atlassian.net/browse/LR-387

cc: @reshmi-nair @rhwarrier @Krishnaj20 @vinukumar-vs @gandham-santhosh
